src/lib/Zikula/Bundle/CoreBundle/Twig/Extension/CoreExtension.php

```diff
@@ -362,4 +363,20 @@ public function hasPermission($component, $instance, $level)
 
         return (bool) $result;
     }
+    
+    /**
+     * @param string $modname
+     * @param bool $force
+     * @return bool
+     */
+    public function modAvailable($modname = null, $force = false)
+    {
+        if (empty($modname)) {
```

Comment:
It is set to NULL as a default, to ensure the a module name is always passed. There isn't a specific module name to set as the default.